WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -323,7 +323,10 @@ private extension TotalsView {
             case .validatingOrderError:
                 return .outlined
             case .paymentError:
-                return .topAligned
```

Comment:
I think we can remove the `topAligned` definition if we're not using it here. Or update it to match this new one.

Reply:
Good catch, removed in https://github.com/woocommerce/woocommerce-ios/pull/15249/commits/5ef6ccfa1f78518e4214044ca33ae0cbee149676.